theano/tensor/signal/downsample.py

```diff
@@ -29,8 +29,8 @@ def max_pool_2d(input, ds, ignore_border=False):
     :param input: input images. Max pooling will be done over the 2 last
         dimensions.
     :type ds: tuple of length 2
-    :param ds: factor by which to downscale (vertical ds, horizontal ds). 
-        (2,2) will halve the image in each dimension. 
+    :param ds: factor by which to downscale (vertical ds, horizontal ds).
+        (2,2) will halve the image in each dimension.
```

Comment:
The method max_pool_2d need an st parameter. I'll fix it.

Also the GPU opt that move this op the GPU do not take check the new parameter! This introduce a bug. I'll fix it too.